src/project-tests/pomodoro-clock-tests.js

```diff
@@ -78,476 +84,718 @@ export default function createPomodoroClockTests() {
                 resetTimer();
-                const timerLabelAfterReset = document.getElementById('timer-label').innerText;
-                const secondsAfterReset = getSeconds(document.getElementById("time-left").innerHTML);
+                const timerLabelAfterReset = document.getElementById(
+                  'timer-label'
+                ).innerText;
+                const secondsAfterReset = getSeconds(
+                  document.getElementById('time-left').innerHTML
+                );
 
                 // see if timer label changed back
                 if (orignalTimerLabel !== timerLabelAfterReset) {
-                  reject(new Error("Default timer label was not properly reset"));
+                  reject(
+                    new Error('Default timer label was not properly reset')
+                  );
                 }
 
-                // wait another 1.5 seconds to be sure value has not changed (pomodoro is stopped)
-                setTimeout(_ => {
-                  const breakLengthAfterResetIsCorrect = document.getElementById("break-length").innerHTML == 5;
-                  const sessionLengthAfterResetIsCorrect = document.getElementById("session-length").innerHTML == 25;
-                  if (!breakLengthAfterResetIsCorrect || !sessionLengthAfterResetIsCorrect) {
-                    reject(new Error("Default values for break length and session length were not properly reset"));
+                // wait another 1.5 seconds to be sure value has not changed 
+                // (pomodoro is stopped)
+                setTimeout(() => {
+                  const breakLenAfterResetCorrect = (
+                    document.getElementById('break-length').innerHTML === '5'
+                  );
+                  const sessionLenAfterResetCorrect = (
+                    document.getElementById('session-length').innerHTML === '25'
+                  );
+                  if (
+                    !breakLenAfterResetCorrect || !sessionLenAfterResetCorrect
+                  ) {
+                    reject(
+                      new Error('Default values for break length and session' +
+                      'length were not properly reset')
+                    );
                     return;
                   }
-                  const secondsAfterAWhile = getSeconds(document.getElementById("time-left").innerHTML);
-                  if (secondsAfterReset === secondsAfterAWhile)
+                  const secondsAfterAWhile = getSeconds(
+                    document.getElementById('time-left').innerHTML
+                  );
+                  if (secondsAfterReset === secondsAfterAWhile) {
                     resolve();
-                  else
-                    reject(new Error("Pomodoro has paused but time continued elapsing"));
+                  } else {
+                    reject(
+                      new Error('Pomodoro has paused but time continued' +
+                      'elapsing')
```

Comment:
Please put the parameter on it's own line. And am I missing something? Does this statement need a semi-colon at the end? I'm surprised ESLint didn't warn about that.